kitsune/sumo/static/sumo/less/main.less

```diff
@@ -1905,11 +1928,6 @@ input[type=button],
   z-index: 1000;
 }
 
-/* Survey Gizmo bottom bar */
-.sg-b-bar {
-  box-shadow: 0px -3px 3px rgba(0, 0, 0, 0.1);
-}
```

Comment:
This change is needed. This is one of the reasons why I don't like rebasing: it can easily wipe out newer changes if you aren't careful, and forces the burden of the merge conflicts on you instead of me.